packages/ds-react-core/tsconfig.build.json

```diff
@@ -7,7 +7,17 @@
     "declarationDir": "dist/types",
     "declarationMap": true,
     "sourceMap": true,
-    "skipLibCheck": false
+    "skipLibCheck": false,
+    "types": ["node", "react", "react-dom"]
   },
-  "exclude": ["src/**/*.stories.ts", "src/**/*.stories.tsx", ".storybook"]
+  "exclude": [
+    "src/**/*.stories.ts",
+    "src/**/*.stories.tsx",
+    "src/**/*.test.ts",
+    "src/**/*.test.tsx",
+    "vite.config.ts",
+    "vitest.setup.ts",
+    "vitest.config.ts",
```

Comment:
The vite config file was not previously being type-checked. It, and the new vitest config and setup files, have been included in the root-level tsconfig for type-checking, and explicitly excluded from the build config.